tests/Moq.Tests/MockRepositoryFixture.cs

```diff
@@ -303,5 +313,12 @@ public virtual void BaseMethod()
 				BaseCalled = true;
 			}
 		}
+
+		public class ConcreteClass : BaseClass
```

Comment:
Since this class will potentially be shared with / reused by other tests in the future, could you please rename it to something more specific, like e.g. `ClassWithoutParameterlessConstructor`? Thanks!

Reply:
You resolved this review comment, but didn't actually change the class' name. Instead, you changed the name of the *other* `ConcreteClass` (in `IssueReportsFixture.cs`) to `ClassWithoutParameterlessConstructor` even though it has a default constructor...

(P.S. on a side note, in future PRs, I'd appreciate it if you didn't resolve review comments yourself; let the reviewers perform that step. It should be up to the original author of a review comment to decide whether their comment / request has been sufficiently addressed. Letting reviewers resolve comments makes it easier for them to review your PRs because it will take less time to keep an overview of what remains to be done. :wink:)